### ISSUE TITLE
Error thrown when launching multiple tables

### ISSUE THREAD

**Issue by arbitrager44** [dmnfarrell/pandastable#130]
@dmnfarrell: Within my application, I have a main `Table` created at the outset, and later a second table may be created based on what the user does. In testing I'm finding if I launch the second table, close it, and come back to the main, I'm getting this error:

>_tkinter.TclError: invalid command name ".!toplevel3.!frame2.!tableovrride"

Note that I've created a class called `TableOvrRide` which launches pandastable `Table` in the `__init__`:

```
class TableOvrRide(Table):
    #Override standard Table functions
    def __init__(self, parent=None, **kwargs):
        Table.__init__(self, parent, **kwargs)
```
I then have a bunch of other functions I wanted to run along the `Table` class. Is this invalid within pandastable? If not, do you have any idea what the error above is telling me? Thanks in advance:

**Comment by dmnfarrell** [comment]
The error means it's trying to call a function on a destroyed widget I think. You could override the close() method in the sub-class and see of that helps?

**Comment by arbitrager44** [comment]
OK in trying to cause the error in as many fun and creative ways possible... I think I've narrowed this down to the following. I have a function that overrides `handleCellEntry` using a `super()` at its outset. The extended error message from Tkinter suggests the problem is triggered by the `get_row_clicked` function within pandastable:

>Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\arbit\AppData\Local\Programs\Python\Python36-32\lib\tkinter\__init__.py", line 1699, in __call__ 
return self.func(*args)
  File "C:\Users\arbit\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pandastable\core.py", line 2219, in handle_arrow_keys
    row = self.get_row_clicked(event)
  File "C:\Users\arbit\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pandastable\core.py", line 1915, in get_row_clicked
    y = int(self.canvasy(event.y))
  File "C:\Users\arbit\AppData\Local\Programs\Python\Python36-32\lib\tkinter\__init__.py", line 2457, in canvasy
    self._w, 'canvasy', screeny, gridspacing))
_tkinter.TclError: invalid command name ".!toplevel2.!frame2.!tableovrride"

So it seems most likely to me that when `handleCellEntry` is called via typing into Table1, for some reason my code still thinks it is working with Table2 (or more specifically in `toplevel2`). Since I've destroyed Table2, it is throwing this error. 

I'm still working on the solution, just wanted to post more detail in case it is helpful to others.

**Comment by dmnfarrell** [comment]
Yes this seems to be a focus problem. I can't replicate your issue, but something weird happens when u have multiple tables in one app. You can see in some examples I added: https://github.com/dmnfarrell/pandastable/blob/master/examples/simpletests.py

**Comment by arbitrager44** [comment]
Update. If I just `withdraw()` the window with the second table, everything seems to work fine. I suppose the windows are staying in memory, but my application isn't terribly memory intensive so I'm not real worried about it.

FWIW, I tried using a `<Destroy>` binding initially but that doesn't work. It seems tkinter does the destroying before whatever intercept function I assign to the binding. Live and learn. Instead I did:

`self.toplvl.protocol('WM_DELETE_WINDOW', lambda: self.toplvl.withdraw())`
